docs/resources/azurerm_key_vault_key.md.erb

```diff
@@ -0,0 +1,128 @@
+information see the [Official Azure Documentation](https://docs.microsoft.com/en-us/rest/api/keyvault/getkey/getkey).
+
+At the moment, there doesn't appear to be a way to select the version of the
+Azure API docs. If you notice a newer version being referenced in the official
+documentation please open an issue or submit a pull request using the updated
+version.
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+This resource first became available in 1.3.0 of the inspec-azure resource pack.
+
+## Syntax
+
+The `vault_name` and `key_name` must be given as a parameter. 
+
+You may also specify a `key_version` - if no version is specified, the most recent version of the key will be used.
+
+    describe azurerm_key_vault_key('vault-101', 'secret-key') do
+      its('key.kid')            { should_not be_nil }
+      its('attributes.enabled') { should eq true }  
+    end
+
+<br />
+
+## Examples
+
+If a Key Vault Key is referenced with a valid `Vault Name`, `Key Name` and `Key Version`
+
+    describe azurerm_key_vault_key('vault-101', 'secret-key', '7df9bf2c3b4347bab213ebe233f0e350') do
+        it                        { should exist }
+        its('key.kid')            { should_not be_nil }
+        its('attributes.enabled') { should eq true }
+    end
+
+If a Key Vault Key is referenced with an invalid `Vault Name` or `Key Name`
+
+    describe azurerm_key_vault_key('vault-101', 'incorrect-key') do
+        it { should_not exist }
+    end
+
+<br />
+
+## Parameters
+
+  - `vault_name`  - The name of the Key Vault.
+  - `key_name`    - The name of the Key.
+  - `key_version` (Optional) - The version of a Key, e.g. `7df9bf2c3b4347bab213ebe233f0e350`.
+
+## Attributes
+
+ -   `attributes`
+ -   `key`
+ -   `managed`
+ -   `tags`
+
+### attributes
+The key management attributes.
+
+### key
+The JSON web key.
+
+### managed
+True if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed will be true.
+
+### tags
+Resource tags applied to the Key.
+
+### Other Attributes
+
+There are additional attributes that may be accessed that we have not
+documented. Please take a look at the [Azure documentation](##-Azure-REST-API-version).
+Any attribute in the response may be accessed with the key names separated by
+dots (`.`).
+
+The API may not always return keys that do not have any associated data. There
+may be cases where the deeply nested property may not have the desired
+attribute along your call chain. If find yourself writing tests against
```

Comment:
"If [you] find"

Sorry, I've corrected it in the other files just now. Could you amend this one as well?